spec/support/shared_examples/have_http_status.rb

```diff
@@ -0,0 +1,15 @@
+shared_examples 'have http status' do |status, skip_before: false|
+  specify '', skip_before: skip_before do
```

Comment:
can you name the block instead of leaving the single quotes?

Reply:
I did that on purpose cause I think it's a little bit redundant. It would be a copy and paste of the `shared_examples`'s name. Do you still think it would be better to add a description to it?